messaging/src/test/java/org/axonframework/commandhandling/distributed/DistributedCommandBusTest.java

```diff
@@ -29,14 +29,18 @@
 import org.axonframework.messaging.MessageHandler;
 import org.axonframework.messaging.MessageHandlerInterceptor;
 import org.axonframework.monitoring.MessageMonitor;
+import org.axonframework.tracing.TestSpanFactory;
```

Comment:
Nit: I would expect the copyright notice to be updated.

Reply:
changed